lib/api/BackbeatAPI.js

```diff
@@ -370,6 +374,17 @@ class BackbeatAPI {
         });
     }
 
+    /**
+     * Collects metrics from Prometheus variables and register for response
+     * @param {*} details - route details from lib/api/routes.js
```

Comment:
Instead of using the asterisk in the braces, can you switch to using `Object` and `Function` respectively for these params to keep uniform within the code?